src/data-loader.ts

```
@@ -1384,6 +1398,42 @@ if (import.meta.vitest != null) {
 			expect(formatDate('2024-12-31T23:59:59Z')).toBe('2024-12-31');
 		});
 
+		it('respects timezone parameter', () => {
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)